### DISCUSSION TITLE
run container with TC Desktop's embedded runtime from command line

### DISCUSSION THREAD

**Discussion by cgeisel** [testcontainers/testcontainers-java#8945] [Q&A] [answered]
Hi folks,

This is probably a strange use case and possibly a dumb question. My organization does not allow the use of Docker Desktop, preferring Podman Desktop. Podman Desktop on Mac does not support LocalStack. I've been using TC Desktop and LocalStack with the embedded runtime and it works great.

However there's a use case that our developer community wants to be able to use LocalStack for, which is running terraform. I have a hacky way to do this, which is to use the "freeze containers shutdown" feature in TC Desktop, then run a dummy test that starts a LocalStack container. Then I can run tflocal commands against the service, but this is not a great workflow.

My question: is there a way to start a container from the command line using the embedded runtime?

**Comment by kiview** [comment, marked as answer]
Sharing from the Slack thread, you can use the regular Docker CLI together with the embedded runtime.
* Select the TCD Docker context: `docker context use tcd`
* The Docker CLI can be installed standalone, e.g. via [Homebrew](https://formulae.brew.sh/formula/docker)

**Comment by cgeisel** [comment]
Thank you!
